frontend/src/metabase/embedding-sdk/types/components/data-picker.ts

```diff
@@ -0,0 +1,18 @@
+import type Database from "metabase-lib/v1/metadata/Database";
```

Comment:
The type needs to be in OSS folder because it's used in OSS when defining the plugin

https://github.com/metabase/metabase/blob/7f790fe68ca323340b207d4cbbdfa2e3a81ce0cc/frontend/src/metabase/plugins/index.ts#L539

At first, I placed the type close to the component, but then a linter complained.